plugins/reporters/static-html/src/main/kotlin/TablesReportModelMapper.kt

```diff
@@ -39,65 +39,6 @@ import org.ossreviewtoolkit.reporter.ReporterInput
  */
 internal object TablesReportModelMapper {
     fun map(input: ReporterInput): TablesReport {
-        val analyzerResult = input.ortResult.analyzer?.result
```

Comment:
*Why* is this being done? For a planned reuse? Or just for a better overview / to reduce nesting?